scripts/epa/ejscreen/test_data/test_data_expected.csv

```diff
@@ -0,0 +1,31 @@
+FIPS,DSLPM,CANCER,RESP,OZONE,PM25,year
+dcid:geoId/21100003002,0.26097804865,9.90183182238678,0.13234899350687,,,2020
+dcid:geoId/60379106013,0.1455270378,27.4107366808536,0.378199239369934,64.378616993464,7.89634575342466,2020
+dcid:geoId/60730083281,0.288497048,32.3673222675371,0.439177057507828,43.5690928104575,9.355618904109592,2020
+dcid:geoId/60730100132,0.365222655,43.68787854391871,0.529720646993164,40.6484725490196,9.304486849315072,2020
+dcid:geoId/120530409082,0.301767118,31.2945093046096,0.456494586734142,33.7503052287582,8.32681452054794,2020
```

Comment:
geoId/60730100132 doesn't seem right.  Is this missing the zero fill fix?

Reply:
Katherine updated the script to add the 0s!